### PR TITLE
publish the cl_khr_device_uuid extension spec

### PR DESCRIPTION
**Pull request by bashbaug**
Publishing as agreed on the OpenCL working group mailing list.

### CODE REVIEW

**Review by nikhiljnv — COMMENTED** [review]
Thanks Ben !! Looks good to me !!

**Review by nikhiljnv — APPROVED** [review]
Approving.

File: ext/cl_khr_device_uuid.asciidoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+// #define CL_DEVICE_LUID_KHR          0x106D
+// #define CL_DEVICE_NODE_MASK_KHR     0x106E
+// ----
+// 
+// Constants describing the size of the driver and device UUIDs, and the device LUID:
+// 
+// [source,c]
+// ----
+// #define CL_UUID_SIZE_KHR            16
+// #define CL_LUID_SIZE_KHR            8
```
Comment (alycm) [inline ext/cl_khr_device_uuid.asciidoc:43]: These constants aren't in the XML so the headers will be the only written source for their values (once they're added there).

The `cl_khr_device_uuid` entry could have a `<require comment="Constants">` section.

**Review by alycm — COMMENTED** [review]
LGTM, just one small XML point.

File: ext/cl_khr_device_uuid.asciidoc (hunk 1 above)
Comment (bashbaug) [reply ext/cl_khr_device_uuid.asciidoc:43]: The XML change is pretty straightforward - I'll add that and include it in this PR.

Do we want to describe the **values** of these constants here in the spec though, or only in the XML and headers?  We do include the values of several other constants in the spec, but they're mostly numerical limits in the half-precision extension.  We don't include the values of other enumerants (though we have talked about adding them).

**Comment by bashbaug** [comment]
Discussed in the September 1st teleconference.  We will make the XML changes in a subsequent PR.

**Pull request merged**